### PR TITLE
New recipe: pbsim (PacBio reads simulator)

### PR DESCRIPTION
**Pull request by marcelm**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by marcelm** [comment]
@bioconda/core A new recipe, please review!

File: recipes/pbsim/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  skip: False
+
+test:
+  files:
+    - reads.fastq
+    - ref.fasta
+  commands:
+    # There does not seem to be a way to get an exit code of 0 from the program
+    # without actually simulating something.
+    - pbsim --length-min 5 --sample-fastq reads.fastq ref.fasta
```
Comment (johanneskoester) [inline recipes/pbsim/meta.yaml]: You could also grep from the help message. We do this e.g. for delly.

File: recipes/pbsim/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+source:
+  url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pbsim/pbsim-1.0.3.tar.gz
+
+requirements:
+  build:
+    - gcc
+
+build:
+  skip: False
```
Comment (johanneskoester) [inline recipes/pbsim/meta.yaml]: Section can be left out.

File: recipes/pbsim/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  home: https://code.google.com/archive/p/pbsim/
+  license: GPLv2
+  summary: "PBSIM simulates PacBio reads"
+
+source:
+  url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pbsim/pbsim-1.0.3.tar.gz
+
+requirements:
+  build:
+    - gcc
```
Comment (johanneskoester) [inline recipes/pbsim/meta.yaml]: Did you check that no linking against libgcc happens? You can do so with conda inspect.

File: recipes/pbsim/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  name: pbsim
+  version: "1.0.3"
+
+about:
+  home: https://code.google.com/archive/p/pbsim/
+  license: GPLv2
+  summary: "PBSIM simulates PacBio reads"
+
+source:
+  url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/pbsim/pbsim-1.0.3.tar.gz
```
Comment (johanneskoester) [inline recipes/pbsim/meta.yaml]: Can you please add an md5 checksum?

**Comment by marcelm** [comment]
Thanks, I changed the recipe as suggested.

**Comment by marcelm** [comment]
I assume I can merge it now. Please complain if I should have waited longer.

File: recipes/pbsim/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  md5: c070394d42b6e4cdae0bb2aaf4856782
+
```
Comment (johanneskoester) [inline recipes/pbsim/meta.yaml]: It is in the guidelines. We had gcc removed some time ago, but now we share the build environment with conda forge, and they haven't yet made this step. If you find the time, please add gcc and also libgcc.

**Pull request merged**